chia/types/spend_bundle.py

```diff
@@ -21,9 +21,13 @@ class SpendBundle(Streamable):
     between transactions are more flexible than in bitcoin).
     """
 
-    coin_spends: List[CoinSpend]
+    coin_solutions: List[CoinSpend]
     aggregated_signature: G2Element
 
+    @property
+    def coin_spends(self):
```

Comment:
Questions I have when I see this:

- does this affect the binary serialization? (I'm pretty sure it doesn't)
- does this affect the JSON serialization? (this one I'm less familiar with)

Using `@property` here looks reasonable otherwise.

Reply:
I don't know about #1 but I can test it, but no to #2 this makes the JSON serialization come out with a "coin_solutions" key no matter what.

Reply:
Update, I tested a random spend bundle with and without this property and the serialization formats were consistent for both